src/components/PeopleInfo/PeopleInfo.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Person } from '../../types/Person';
+
+type Props = {
```

Comment:
use interface prefer.
Types have unique features but the interface is best for scaling.